cmd/setup-gh.go

```diff
@@ -99,8 +100,8 @@ func fillSetUpConfig(sc *providers.SetUpCmd) {
 
 	if sc.SubscriptionID == "" {
 		if strings.ToLower(sc.Provider) == "azure" {
-			currentSub := providers.GetCurrentAzSubscriptionId()
-			sc.SubscriptionID = GetAzSubscriptionId(currentSub)
+			currentSubs := providers.GetCurrentAzSubscriptionId()
+			sc.SubscriptionID = GetAzSubscriptionId(currentSubs)
```

Comment:
since this will now return a string with the name and the subscriptionID, we'll need to probably make a struct to hold both the label and the subscriptionId so we can pass them out, otherwise this line would set the `sc.SubscriptionId` to the whole label ex: `MySubscriptionName 1122334455` which won't be parsed correctly in the other functions